src/linter/linterReporting.ts

```diff
@@ -0,0 +1,44 @@
+export const MESSAGES = {
+	SHORT__NO_DIRECT_DATATYPE_ACCESS: "Deprecated access to DataType ('{0}').",
```

Comment:
This is the only message that ends with a period (`.`). As we didn't use a period at the end, I'd suggest to also not use one here.

Reply:
Yep, I didn't want to "break" the snapshots, but will update it